dockers/docker-orchagent/supervisord.conf

```diff
@@ -83,6 +83,18 @@ stderr_logfile=syslog
 dependent_startup=true
 dependent_startup_wait_for=swssconfig:exited
 
+[program:coppmgrd]
+command=/usr/bin/coppmgrd
+priority=6
+autostart=false
+autorestart=unexpected
```

Comment:
@dgsudharsan For the process `coppmgrd`, the `autorestart` field was set to `unexpected`. That means if it exited unexpectedly, this process will be restarted automatically. So I am wondering whether we should put this process in the `critical_processes` or not since the processes in that file can not be restarted if they exited?